packages/now-python/test/fixtures/13-wsgi-url-params-encoding/now.json

```diff
@@ -1,15 +0,0 @@
-{
-  "version": 2,
-  "builds": [{ "src": "*.py", "use": "@vercel/python" }],
-  "routes": [{ "src": "/%E6%82%A8%E5%A5%BD", "dest": "custom.py" }],
-  "probes": [
-    {
-      "path": "/?%E6%82%A8%E5%A5%BD=/",
-      "mustContain": "path:  query: {'您好': '/'}"
```

Comment:
These tests were actually wrong, as compared against other languages like Node.js and against actual Python server implementations (as shown in the new test case in this PR).